consensus/src/epoch_manager.rs

```diff
@@ -425,6 +426,7 @@ impl EpochManager {
         } else {
             self.commit_state_computer.clone()
         };
+        state_computer.new_epoch(&epoch_state);
```

Comment:
just use self.commit_state_computer.new_epoch() so we don't need to add this for ordering state computer